src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger.java

```diff
@@ -251,7 +245,7 @@ public void doRun(Integer prNumber) {
                 cause.setPollingLog(pollingLogAction.getLogFile());
                 build(cause);
             } catch (IOException e) {
-                LOGGER.error("can't trigger build {}", e.getMessage());
+                LOGGER.error("can't trigger build {}", e.getMessage(), e);
```

Comment:
most of this "good" messages is `can't trigger build null`, so i think first time it should be more verbosed, then it can be turned back